vtex-apps/store-theme
# Make the minicart open when a product is added to it

#### What problem is this solving?

This is to showcase how users can achieve this behavior. 

#### How to test it?

Go to [Workspace](https://victormiranda--storecomponents.myvtex.com/) and add a product to the cart. You should **not** see the traditional toast that comes up today and the minicart should open.

#### Screenshots or example usage:

![Aug-26-2020 13-36-52](https://user-images.githubusercontent.com/27777263/91332237-45346000-e7a2-11ea-9f78-091a0ae13cfb.gif)

#### Related to

- https://github.com/vtex/pixel-manager/pull/29
- https://github.com/vtex-apps/drawer/pull/34
- https://github.com/vtex-apps/minicart/pull/240

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/l4EpkVLqUj8BI7OV2/giphy.gif)
